app/helpers/trace_helper.rb

```diff
@@ -2,4 +2,12 @@ module TraceHelper
   def link_to_tag(tag)
     link_to(tag, :tag => tag, :page => nil)
   end
+
+  def trace_icon(trace, options = {})
+    options[:class] ||= "trace_image"
+    options[:alt] ||= ""
+
+    image_tag trace_icon_path(trace.user, trace),
+              options.merge(:width => 50, :height => 50)
```

Comment:
```suggestion
              options.merge(:size => 50)
```

`user_helper.rb` is partially using `:size`, I think we were moving towards it from `:width`/`:height` for square images.

Reply:
Yes, :size is ok, the resulting css will be the same as before. Changed.